modules/reference/pages/rpk/rpk-cluster/rpk-cluster-license-info.adoc

```diff
@@ -6,6 +6,8 @@ Retrieve license information:
 Organization:    Organization the license was generated for.
 Type:            Type of license: free, enterprise, etc.
 Expires:         Expiration date of the license.
+License Status:  Status of the loaded license (valid, expired, not_present).
```

Comment:
In the preview this shows up as a formatted code block, just want to double check that that should be the case

Reply:
hmm we can remove this format, no strong opinion